Mathlib/Analysis/Convex/StrictConvexSpace.lean

```diff
@@ -233,7 +233,9 @@ theorem not_sameRay_iff_abs_lt_norm_sub : ¬SameRay ℝ x y ↔ |‖x‖ - ‖y
 #align not_same_ray_iff_abs_lt_norm_sub not_sameRay_iff_abs_lt_norm_sub
 
 /-- In a strictly convex space, the triangle inequality turns into an equality if and only if the
-middle point belongs to the segment joining two other points. -/
+middle point belongs to the segment joining two other points.
+
+TODO: Deduplicate from `dist_add_dist_eq_iff_wbtw`. -/
```

Comment:
Can you do this in this PR? I think just eliminating the lemma would be fine.

Reply:
Sure, if you're happy with it.

Reply:
Please do!